assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Footer.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Footer component for SearchFunnelWidget.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
```

Comment:
Also the current year is already 2022 😄. Please, update all your new files to use the correct year in the file headers.

Reply:
Good spot, thanks! That's done :+1: